### DISCUSSION TITLE
Update to rc version?

### DISCUSSION THREAD

**Discussion by wverdi** [danny-avila/LibreChat#3035] [Help Wanted]
Noob question, but i`m new on this haha
Ive been trying to update from 0.7.2 and i wasnt able to do so
ive tried every tutorial ive found on the site to get 0.7.3rc2 running, edited every .yml i found before coming here
is there any other tutorial to do it? 
this my default deploy-compose.yml file, i dont know if i got it right but i was suppose to change the "image" line with the tagged version right?

version: "3.8"
services:
  api:
    # build:
    #   context: .
    #   dockerfile: Dockerfile.multi
    #   target: api-build
    image: ghcr.io/danny-avila/librechat-dev-api:latest
    container_name: LibreChat-API
    ports:
      - 3080:3080
    depends_on:
      - mongodb
      - rag_api
    restart: always
    extra_hosts:
    - "host.docker.internal:host-gateway"
    env_file:
      - .env
    environment:
      - HOST=0.0.0.0
      - NODE_ENV=production
      - MONGO_URI=mongodb://mongodb:27017/LibreChat
      - MEILI_HOST=http://meilisearch:7700
      - RAG_PORT=${RAG_PORT:-8000}
      - RAG_API_URL=http://rag_api:${RAG_PORT:-8000}

**Comment by danny-avila** [comment]
you can use the docker override file to specifically change the image being used without creating potential merge conflicts when updating in the future:

https://www.librechat.ai/docs/configuration/docker_override

Also FYI, you will still see v0.7.2 with latest release candidate version. I will change that in the future.
![image](https://github.com/danny-avila/LibreChat/assets/110412045/3d628475-00ca-4301-9e2d-d8fa7dfb650b)

**Reply by danny-avila** [reply]
in other words, if you are on "latest" it's already the latest, provided you updated: https://www.librechat.ai/docs/local/docker#update-librechat

**Reply by wverdi** [reply]
ooh thanks mate
this was killing me haha

**Reply by danny-avila** [reply]
No worries sorry for the inconvenience this caused you
